build/Main.cs

```diff
@@ -60,7 +60,7 @@
 
 		Directory.CreateDirectory(protosTagDir);
 		Console.WriteLine($"Downloading protos for tag {qdrantVersion} to {protosTagDir}");
-		var url = $"https://api.github.com/repos/qdrant/qdrant/tarball/refs/tags/{qdrantVersion}";
+		var url = "https://api.github.com/repos/qdrant/qdrant/tarball/v1.7.0";
```

Comment:
```suggestion
		var url = $"https://api.github.com/repos/qdrant/qdrant/tarball/{qdrantVersion}";
```
This will use the value defined in `<QdrantVersion>` in Directory.Build.props by default

Reply:
I figured out, that you can set `QdrantVersion = dev` to download protofiles from dev branch and test on dev qdrant docker container

Reply:
QDrant `v.1.7.0` was published this morning. Changed back to qdrant version usage